example/gardener-local/gardenlet/values.yaml

```diff
@@ -92,6 +92,7 @@ config:
         # Those CIDRs must match those specified in the kind Cluster configuration.
         pods: 10.1.0.0/16
         services: 10.2.0.0/16
+        vpn: 10.5.0.0/24
```

Comment:
While I can understand the desire to test the feature in the end-to-end tests it also means that the default case is no longer tested in them. Can we find a middle ground?

Reply:
Good point. To verify both the default case and a non-default configuration, I changed the local seed back to the default configuration and changed the local2 seed to the non-default configuration.
With this, both cases are verified in e2e tests during control plane migrations (modulo https://github.com/gardener/gardener/issues/9604).